Source/NSDictionary+HYPImmutable.h

```diff
@@ -2,10 +2,12 @@
 
 @interface NSDictionary (HYPImmutable)
 
-- (NSDictionary *)hyp_dictionaryByRemovingNullItems;
+- (NSDictionary *)hyp_removingNulls;
 
-- (NSDictionary *)hyp_dictionaryByRemovingKey:(id <NSCopying>)key;
```

Comment:
I make a mistake here, there shouldn''t be a space between `id` and  `<NSCoping>` protocol